<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditorClass</class>
 <widget class="QMainWindow" name="TextEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>843</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>幻梦自由 - GTA4翻译小工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableView" name="table_texts">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_open">
          <property name="text">
           <string>加载</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="text">
           <string>保存(&amp;S)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_prev">
          <property name="text">
           <string>上一条(&amp;A)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_next">
          <property name="text">
           <string>下一条(&amp;Z)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>原文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_original">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>机翻(参考用)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_machine_translated">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>译文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_translated">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_tokens" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="tokens_spacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>870</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>退出程序</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextEditor.qrc"/>
 </resources>
 <connections/>
</ui>
